src/app/components/content/IsaacClozeQuestion.tsx

```diff
@@ -99,7 +99,7 @@ export function IsaacClozeQuestion({doc, questionId, readonly}: {doc: IsaacCloze
     const pageQuestions = useSelector(selectors.questions.getQuestions);
     const questionPart = selectQuestionPart(pageQuestions, questionId);
     const currentAttempt = questionPart?.currentAttempt as (ItemChoiceDTO | undefined);
-    const cssFriendlyQuestionPartId = questionPart?.id?.replace(/\|/g, '-') ?? ""; // Maybe we should clean up IDs more?
+    const cssFriendlyQuestionPartId = (questionPart?.id || doc.id)?.replace(/\|/g, '-') ?? ""; // Maybe we should clean up IDs more?
```

Comment:
`questoinId` is what we use to filter out the `questionPart` from the registered questions so it's safe to just use `questionId` directly. The reason this did not work on Tests/Quizzes was that Tests/Quizzes do not register their question parts. 
```suggestion
    const cssFriendlyQuestionPartId = questionId?.replace(/\|/g, '-') ?? ""; // Maybe we should clean up IDs more?
```